taxonomies/ingredients.txt

```diff
@@ -100,6 +100,12 @@ stopwords:sl:vsebuje, sestavine
 stopwords:sv:av, från, med, innehåller, inklusive, bl a, bland annat, minimum, maximum, varav, certifierad, tillsatt, valio
 stopwords:zh::配料
 
+# Eurocode properties
+# e.g.
+# eurocode_2_group_2:en: 8.25
+# eurocode_2_group_3:en: 8.25.50
+# are from https://web.archive.org/web/20170712020557/http://www.ianunwin.demon.co.uk/eurocode/docmn/ec99/ecmgintr.htm#MGList
+
```

Comment:
do you have a spreadsheet somewhere where you track associations if you need help ?

Reply:
I don't have a spreadsheet, we could create one